public/app/plugins/datasource/cloud-monitoring/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -22,6 +23,7 @@ export class ConfigEditor extends PureComponent<Props> {
           <InlineField label="Type" labelWidth={20}>
             <Select
               menuShouldPortal
+              aria-label="Type"
```

Comment:
Here we can use `htmlFor` on `InlineField`, and `inputId` on `Select` so we don't need to repeat the label.
See influxdb's `ConfigEditor.tsx` for an example